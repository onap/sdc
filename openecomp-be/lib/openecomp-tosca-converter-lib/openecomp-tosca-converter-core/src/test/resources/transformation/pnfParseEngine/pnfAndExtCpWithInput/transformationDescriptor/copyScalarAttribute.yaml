- transformation-for: nodeTemplate
  query:
    type: tosca.nodes.nfv.PnfExtCp
  conversions:
    - query:
        properties:
          trunk_mode:
      to-name: trunk_mode_copy
      to-value:
        strategy: copy