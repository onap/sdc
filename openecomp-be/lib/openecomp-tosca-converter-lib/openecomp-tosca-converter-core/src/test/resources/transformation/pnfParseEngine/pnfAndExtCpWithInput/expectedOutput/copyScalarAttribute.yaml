topology_template:
  node_templates:
    pnfExtCp_1:
      properties:
        trunk_mode_copy: false